src/styles/blocks/header.scss

```diff
@@ -0,0 +1,14 @@
+.header {
+  padding: 0 50px;
+
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+
+  box-shadow: 0 2px 4px rgba(0, 0, 0, 0.05);
+
+  &__logo {
+    width: 40px;
+    height: 40px;
```

Comment:
also, you can create a variable in case you change the size

Reply:
done